Arrt/Widgets/ColorPicker.cpp

```diff
@@ -197,7 +198,8 @@ static void drawCheckerboardPattern(QPainter& painter, const QRect& rect)
 void ColorPicker::paintEvent(QPaintEvent*)
 {
     QColor c = m_color;
-    QRect r = rect().adjusted(0, 0, -1, -1);
+    const int borderW = ArrtStyle::s_focusedControlBorderWidth + 1;
+    QRect r = rect().adjusted(borderW, borderW, -borderW, -borderW - 1);
```

Comment:
I assume the asymmetry here is deliberate (i.e. the -1 applies only to one of the components).